### DISCUSSION TITLE
How to treat dead endpoints?

### DISCUSSION THREAD

**Discussion by idleberg** [badgen/badgen.net#584] [Q&A]
Some years ago, I've added the support for Atom badges (strictly speaking that's `apm` badges for Atom Package Manager). In December 2022, GitHub has shut down the entire infrastructure for its Atom editor, including the endpoints that provide the data for the `apm` badges. What is the preferred way to deal with this? The way I see it, there are two ways:

1. graceful: badges will be updated to reflect the status of the project (e.g. "discontinued")
2. simply remove the badges (what will this result in for users of those badges)

Once this has been decided, I can make those changes. Further, I'd like to add new badges for one of the community-maintained forks of Atom.

**Comment by amio** [comment]
I think we can update the status to "discontinued", also put a deprecation note on the docs page
